src/test/mediapipe/python/python_backend.pbtxt

```diff
@@ -0,0 +1,43 @@
+#
```

Comment:
Please name test files with description what does it do - here it could be: two_python_outputs_graph.pbtxt